api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/redis/BaseRedisProperties.java

```diff
@@ -183,6 +192,7 @@
      */
     @Getter
     @Setter
+    @ToString
```

Comment:
Please remove all unnecessary ToString methods.

Reply:
This is needed for [this change](https://github.com/apereo/cas/blob/35238550b84ae108e2c0c6f8a988f3096b2a44ad/support/cas-server-support-redis-core/src/main/java/org/apereo/cas/redis/core/RedisObjectFactory.java#L64). I added it to show the configuration at launch.

Reply:
removed